### ISSUE TITLE
Missing qemu-user-static-x86-8.2.4-1.fc40.aarch64.rpm  on kojipkgs.fedoraproject.org

### ISSUE THREAD

**Issue by adrianriobo** [crc-org/snc#967]
Building arm64 bundles for 4.17.0-rc5 the snc execution failed due to missing qemu static rpm

**Comment by cfergeau** [comment]
qemu-user-static-x86-8.2.4-1.fc40.aarch64.rpm was never released as on official f40 update, and now the build has been deleted https://koji.fedoraproject.org/koji/buildinfo?buildID=2465534
In the future, better to pick a build which is tagged in `f40-updates`, these ones should not go away.

**Comment by cfergeau** [comment]
Might be better to temporarily add/enable a f40-updates repo so that we always get the latest official build instead of hardcoding the URL.
```
$ cat /etc/yum.repos.d/fedora-updates.repo 
[updates]
name=Fedora $releasever - $basearch - Updates
#baseurl=http://download.example/pub/fedora/linux/updates/$releasever/Everything/$basearch/
metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-released-f$releasever&arch=$basearch
enabled=1
countme=1
repo_gpgcheck=0
type=rpm
gpgcheck=1
metadata_expire=6h
gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
skip_if_unavailable=False
```

**Comment by redbeam** [comment]
@praveenkumar yes I can take a look

**Comment by redbeam** [comment]
Fixed in #986
